composer.json

```diff
@@ -25,15 +25,15 @@
     "require": {
         "php": "^7.0",
         "guzzlehttp/guzzle": "^6.0",
-        "psr/container": "^1.0"
+        "psr/container": "^1.0",
+        "zfr/zfr-aws-utils": "^1.1"
```

Comment:
As it’s not needed for people using postmark I’d suggest you move this into the suggested dependencies. This will prevent pulling this and the aws sel for people not using it

Reply:
Oups my bad ;)